PyInstaller/fake-modules/site.py

```diff
@@ -56,3 +56,11 @@ def __call__(self, *args, **kwds):
          # *every* frozen app
          pydoc = __import__(''.join('pydoc'))
          return pydoc.help(*args, **kwds)
+
+try:
+    import __builtin__ as builtins # Python 2
+except ImportError:
+    import builtins # Python 3
+# OSGeo wraps the builtin help function even though it is not available
+# https://github.com/pyinstaller/pyinstaller/issues/1522
+builtins.help = _Helper()
```

Comment:
I'm starting to think the best/cleanest option it to just not support gdal versions affected by this.